src/controllers/User/UserSignInController.ts

```diff
@@ -0,0 +1,37 @@
+import * as jwt from 'jsonwebtoken';
+import * as bcrypt from 'bcrypt';
+
+import config from '../../config';
+import models from '../../models/db';
+
+export const signInUser = async (userInfo) => {
+  const User =  models.user;
+
+  return await User
+    .findOne({
+      where: {
+        username: userInfo.username
+      }
+    })
+    .catch(res => {
```

Comment:
The name of the argument, as in widely used cases, could be 'err', or 'e' in catch(). This can actually help when differentiating with then(). I suggest it be 'err'.